lib/utils.js

```diff
@@ -215,3 +218,16 @@ export const flattenObjectDeep = obj =>
   );
 
 export const stripHTML = htmlContent => sanitizeHtml(htmlContent, { allowedTags: [], allowedAttributes: {} });
+
+export const checkRequiresAddress = values => {
+  const isSelfHostedCollective = values.payee.id === values.payee.host?.id;
```

Comment:
That's not exactly the definition of a self-hosted collective, Fiscal Hosts are also matching this condition.

Reply:
I've retrieved the `isHost` field and checked on that instead so as to make it a bit more direct rather than going with this id approach. Btw, I think it's okay to just check if the collective is a host correct? Regardless of whether it's a self hosted collective or not we will still require the address. 🤔

Reply:
`isHost` is still not the strict definition of a Self Hosted collective, so your change didn't fix the feedback at all.

It was more about the name of the variable which was confusing.

Reply:
>`isHost` is still not the strict definition of a Self Hosted collective, so your change didn't fix the feedback at all.

I am a bit confused with this one. Why do we need a strict definition of a self hosted collective? If the collective is a host, then we need to show the address regardless of whether it's self hosted or not correct? 🤔 

>It was more about the name of the variable which was confusing.

The variable was removed in my latest change.